server/controllers/FollowController.js

```diff
@@ -51,6 +58,18 @@ class FollowController {
             })
               .spread((followed, status) => {
                 if (status) {
+                  // TODO 2: Also add a new entry to notification events
+                  type = 'Follow';
+                  addNewNotificationEvent(
+                    'follows',
+                    authorId,
+                    id,
+                    'You have a new follower/unfollower',
```

Comment:
This message is not specific.
 I don't know if he or she is a followed me or unfollowed me.
Please make it more descriptive

Reply:
Feedback implemented @augustineezinwa